### PR TITLE
[PATCH tracking] Artwork page tracking QA

### PR DESCRIPTION
**Pull request by damassi**
Tracking qa from @cavvia

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 225794c985f96e63df525362ff285e211054d7bd

https://deploy-preview-1555--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 211de2079fe1708810a1067306a29f7b4932bf40

https://deploy-preview-1555--artsy-reaction.netlify.com

File: src/Apps/Artwork/Components/ArtistInfo.tsx [hunk 1]
```diff
@@ -67,6 +68,15 @@ export class ArtistInfo extends Component<ArtistInfoProps> {
                   <FollowArtistButton
                     artist={this.props.artist}
                     user={user}
+                    trackingData={{
+                      modelName: Schema.OwnerType.Artist,
+                      context_module: [
+                        Schema.ContextModule.Sidebar,
+                        Schema.ContextModule.Biography,
+                      ],
+                      entity_id: _id,
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/ArtistInfo.tsx]: 👍

File: src/Apps/Artwork/Components/ArtistInfo.tsx [hunk 2]
```diff
@@ -102,7 +112,9 @@ export class ArtistInfo extends Component<ArtistInfoProps> {
                   <Spacer mb={1} />
                   <ArtistBio
                     bio={this.props.artist}
-                    onReadMoreClicked={this.trackArtistBioReadMoreClick}
+                    onReadMoreClicked={this.trackArtistBioReadMoreClick.bind(
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/ArtistInfo.tsx]: oh yea totally, i usually do `onClick={() => this.onClick` as well (to avoid `bind(this)` language).

**Comment by mzikherman** [comment]
Looks good!

**Pull request merged**